docs/HowTo/Get-Started/Connect-To-Testnet.md

```diff
@@ -0,0 +1,184 @@
+
+Teku does not contain a preset for Testnet 5. The necessary configuration can be found in the [eth2-testnets repository on Github](https://github.com/eth2-clients/eth2-testnets/tree/master/lighthouse/testnet5).
+
+## Enable an ETH1 Connection
+
+**Prerequisites**:
+
+* [Install Hyperledger Besu](https://besu.hyperledger.org/en/latest/HowTo/Get-Started/Install-Binaries) Ethereum 1.0 client in addition to the Teku Ethereum 2.0 client.
+
+The deposit contract for Schlesi and Topaz are deployed to the _Goerli_ ETH1 testing network. To enable an ETH1 connection on Teku, configure Hyperledger Besu to connect to the Goerli testnet and expose the RPC-HTTP APIs:
+
+```bash
+besu --network=goerli \
+--rpc-http-enabled=true \
+--rpc-http-port=8545 \
+--rpc-http-api=ETH,NET,WEB3
+```
+
+Once you have a local Goerli node running with an activated RPC-HTTP API, you can connect Teku to it by enabling `--eth1-enabled`.
+
+!!! example
+
+    ```
+    teku --network=schlesi --eth1-enabled --eth1-endpoint=localhost:8545
+    ```
+
+!!! tip
+
+    See also: [Hyperledger Besu - Run a node on Goerli testnet](https://besu.hyperledger.org/en/stable/HowTo/Get-Started/Starting-node/#run-a-node-on-goerli-testnet)
+
+## Boostrap a new Testnet
+
+If you already have created a custom testnet or plan to do so, Teku allows you to pass different configurations to the beacon-chain node. A custom network requires a couple of things:
+
+* A network configuration that can be either one of the `minimal` or `mainnet` presets, or a custom `yaml` file containing the network specification.
+* An `ssz` genesis state or information about the ETH1 deposit contract.
+* Eventually bootnodes to allow other users to connect to your network.
+
+**Network configuration**:
+
+The network configuration can be either of the presets `minimal` and `mainnet` or a `yaml` file. The custome configuration file can be passed as local file or remote config fetched via HTTP, e.g.:
+
+* `--network="minimal"`
+* `--network="mainnet"`
+* `--network="./path/to/chain.yaml"`
+* `--network="https://github.com/goerli/schlesi/raw/master/teku/chain.yaml"`
+
+!!! example
+
+    ```
+    teku --network="https://github.com/goerli/schlesi/raw/master/teku/chain.yaml"
+    ```
+
+**Deposit Contract**:
+
+The deposit contract is a deployed instance of the [`validator_registration.vy`](https://github.com/ethereum/eth2.0-specs/blob/dev/deposit_contract/contracts/validator_registration.vy) on an ETH1 network of your choice.
+
+`--eth1-deposit-contract-address="0xaa888248144bc5d584a7f400839d0d912f21c39a"`
+
+!!! tip
+
+    See also: [How to run your own Beacon Chain](https://dev.to/q9/how-to-run-your-own-beacon-chain-e70)
+
+Note, that reading the deposit contract requires an enabled ETH1 connection, e.g.:
+
+`--eth1-enabled --eth1-endpoint="localhost:8545"`
```

Comment:
only one example line may be ok. I would prefer you to keep only the second full one.